src/main/kotlin/com/chattriggers/ctjs/minecraft/wrappers/Player.kt

```diff
@@ -34,6 +35,11 @@ object Player {
         //#endif
     }
 
+    @JvmStatic
+    fun getTeam(): ScorePlayerTeam? {
```

Comment:
Seems like this should have a wrapper if we're adding a utility method to retrieve it.